### PR TITLE
The Newsweek clone page.

### PR DESCRIPTION
**Pull request by Godswilly**
I tried to clone the Newsweek website with HTML, CSS, Bootstrap, and media queries. I also tried to make the layout using Bootstrap inbuilt properties. The site is responsive to different screen sizes.

### CODE REVIEW

**Comment by Godswilly** [comment]
This is the Newsweek web page clone. I tried to work with Bootstrap and media queries to make the site responsive.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2141 @@
...
+          <a class="nav-sport  text-dark font-weight-bold text-nowrap" href="#">The Debate</a>
+        </li>
+
+        <li class="nav-item border-right">
+          <a class="nav-sport  text-dark font-weight-bold" href="#">Vantage</a>
+        </li>
+
+        <li class="nav-item pr-xl-5 mr-xl-5">
+          <a class="nav-sport  text-dark font-weight-bold text-nowrap" href="#">Weather
+            <span>Search <i class="fas fa-search nav-item"></i> </span></a>
```
Comment (andresporras3423) [inline index.html]: The search in the navbar should be an input

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @Godswilly** 
Good job with your first submission for this project :muscle:  
your page looks really similar to the original :clap: :clap: :clap:  
Your project passes most of the requirements. Just a few details to improve before you can go ahead:
- [ ]  Use input for the search in the navbar
- [ ] For the large screen size, burger icon shouldn't appear
- [ ]  For the medium screen size, the user icon shouldn't appear. Look at the original page:



And your page:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hi @Godswilly 👋. Your project looks really great! And almost looks like an exact clone 👏👌!

### But we need to work on some changes before we can move ahead.

- Your PR is empty. Please provide a description about what you have done in the project.

- Please add a margin on either side of the whole clone.

#### Clone
image

#### Original
image

- Please make this look like the original.
image

#### Original
image

- Please add some padding to these elements 👇.
image


- Please remove the hamburger icon from the desktop mode. It should appear in tablet mode.
image

#### Original
image

- Please fix this 👇
image

- Decrease the space where I have marked with arrows.
image

- Please fix the squeezing images 👇
image

- Please make this section look like the original.
image

#### Original
image

- The navbar should be fixed in small screens.

#### Clone
Annotation 2020-07-04 132919

#### Original
Annotation 2020-07-04 133051


Apply these changes and request another review.

Happy coding 👨‍💻

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello @Godswilly 👋, it's me again 😀. You are almost there.

- I see that you have tried to add a margin on both sides of your clone. But it's not behaving like the original. In the original, the page maintains a `max-width` of `1300px`. Right now your clone maintains the margin up to `1400px`. Please fix this.

- The search in the navbar should be an input field.
image

- Please fix this to match the original 👇
   image

   #### Original
   image

### Please apply these changes and request another review.

Happy coding 👨‍💻

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle:

### Hi @Godswilly :+1: 

### Your project is getting better and better, there some changes that are required:
- as requested by the previous TSE, the page should have a max width of 1300px,
clone:
Screenshot from 2020-07-04 21-44-20
original:
Screenshot from 2020-07-04 21-45-11
This is two pictures are for 3600 px width.
### Once done with the changes, please ask for another code review

**Review by HADDADSOHAIB — APPROVED** [review]
### Status: Project Approved :+1: 

### Hi @Godswilly   :hand: 

### You did great with the project, you implemented all changes required with the previous TSEs, you can go on and merge this branch into master.

**Pull request merged**